build/deploy/db_schemas/rid.libsonnet

```diff
@@ -1,19 +1,16 @@
 {
   data:{
-    "000001_create_isa_subs_tables.down.sql": importstr "rid/000001_create_isa_subs_tables.down.sql",
-    "000001_create_isa_subs_tables.up.sql": importstr "rid/000001_create_isa_subs_tables.up.sql",
-    "000002_inverted_index.down.sql": importstr "rid/000002_inverted_index.down.sql",
-    "000002_inverted_index.up.sql": importstr "rid/000002_inverted_index.up.sql",
-    "000003_migrate_data_to_index.down.sql": importstr "rid/000003_migrate_data_to_index.down.sql",
-    "000003_migrate_data_to_index.up.sql": importstr "rid/000003_migrate_data_to_index.up.sql",
-    "000004_drop_cells_table.down.sql": importstr "rid/000004_drop_cells_table.down.sql",
-    "000004_drop_cells_table.up.sql": importstr "rid/000004_drop_cells_table.up.sql",
-    "000005_bump_version.down.sql": importstr "rid/000005_bump_version.down.sql",
-    "000005_bump_version.up.sql": importstr "rid/000005_bump_version.up.sql",
-    "000006_add_writer_column.down.sql": importstr "rid/000006_add_writer_column.down.sql",
-    "000006_add_writer_column.up.sql": importstr "rid/000006_add_writer_column.up.sql",
-    "000007_add_index_by_time_subscriptions.down.sql": importstr "rid/000007_add_index_by_time_subscriptions.down.sql",
-    "000007_add_index_by_time_subscriptions.up.sql": importstr "rid/000007_add_index_by_time_subscriptions.up.sql",
-    "000008_db_rename_to_rid.up.sql": importstr "rid/000008_db_rename_to_rid.up.sql"
+    "upto-v1.0.0-remove_isa_subs_tables.sql": importstr "rid/upto-v1.0.0-remove_isa_subs_tables.sql",
```

Comment:
The name of the file seems wrong. Shouldn't it be `upto-v1.0.0-create_isa_subs_tables.sql` ?